package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "chai": "^3.2.0",
     "coveralls": "^2.11.2",
+    "graceful-fs": "4.1.4",
```

Comment:
Ah, interesting. So yeah, this should be better than rolling our own workaround.